x-pack/plugin/otel-data/src/main/resources/index-templates/metrics-service_destination.10m@template.yaml

```diff
@@ -0,0 +1,48 @@
+---
+version: ${xpack.oteldata.template.version}
+index_patterns: ["metrics-service_destination.10m.otel-*"]
+priority: 130
+data_stream:
+  hidden: true
+allow_auto_create: true
+_meta:
+  description: aggregated APM metrics template installed by x-pack
+  managed: true
+composed_of:
+  - metrics@tsdb-settings
+  - otel@mappings
+  - metrics-otel@mappings
+  - semconv-resource-to-ecs@mappings
+  - metrics@custom
+  - metrics-otel@custom
+ignore_missing_component_templates:
+  - metrics@custom
+  - metrics-otel@custom
+template:
+  settings:
+    index:
+      mode: time_series
+  mappings:
+    properties:
+      data_stream.type:
+        type: constant_keyword
+        value: metrics
+      metricset:
+        properties:
+          interval:
+            type: constant_keyword
+            value: 10m
+          name:
+            type: constant_keyword
+            value: service_destination
```

Comment:
`apm-data` only sets `interval`, but not name. I think it's worth setting both - if we decide not to do so, we could merge the 10m and 60m yaml files.

Reply:
Sounds intriguing to be able to reduce the number of index templates. However that would not allow us to have different `@custom` component templates, like `metrics-service_destination.10m.otel@custom`.

Having said that, maybe that level of granularity isn't really needed and what's more important is to have a custom template per interval, such as `metrics-10m.otel@custom`, so that users can set a retention per granularity.

Related to that, I'm wondering whether we should also create a template for the 1m intervals. That would enable us to add a custom component template to manage the retention for the 1m aggregated metrics, as well as having a constant_keyword field for the interval.

Also, I think the proliferation of index templates increases the need for a test that verifies the uniformity and consistency of the index templates. Manually reviewing that isn't really feasible anymore.

Reply:
> Having said that, maybe that level of granularity isn't really needed and what's more important is to have a custom template per interval

Yeah, I agree this would be the right level of granularity. I added those `@custom`s 

> Related to that, I'm wondering whether we should also create a template for the 1m intervals.

Yes! Added that as well.